### ISSUE TITLE
Errors extending javax.swing and java.awt classes

### ISSUE THREAD

**Issue by EotT123** [manifold-systems/manifold#551]
**Describe the bug**
For some reason, extending `javax.swing` and `java.awt` classes doesn't work.
I was going to add extension methods to add some withers for setters methods.

**To Reproduce**

* Example class for `javax.swing`:
```
package extensions.javax.swing.JScrollPane;

import javax.swing.*;
import java.awt.*;

import manifold.ext.rt.api.Extension;
import manifold.ext.rt.api.Self;
import manifold.ext.rt.api.This;

@Extension
public class JScrollPaneExt {

    private JScrollPaneExt() {
        // hide constructor
    }

    public static @Self JScrollPane withViewportView(@This JScrollPane scrollPane, Component view) {
        scrollPane.setViewportView(view);
        return scrollPane;
    }
}
```
When compiling, this results in the following error: `java.lang.NullPointerException: Cannot invoke "manifold.rt.api.util.Pair.getFirst()" because "classSymbol" is null`

----------------

* Example class for `javax.awt`:
```
package extensions.java.awt.Component;

import java.awt.*;
import java.awt.event.MouseListener;

import manifold.ext.rt.api.Extension;
import manifold.ext.rt.api.Self;
import manifold.ext.rt.api.This;

@Extension
public class ComponentExt {

    private ComponentExt() {
        // hide constructor
    }

    public static @Self Component withMouseListener(@This Component component, MouseListener listener) {
        component.addMouseListener(listener);
        return component;
    }
}
```
When compiling, this results in the following error: `Compilation failed: internal java compiler error`

**Desktop (please complete the following information):**
- OS Type & Version:  Windows 10
- Java/JDK version: openjdk 21
- IDE version (IntelliJ IDEA or Android Studio): IntelliJ IDEA 2023.3.4 (Community Edition)
- Manifold version: 2024.1.3 (latest version at the moment)
- Manifold IntelliJ plugin version: 2023.3.15  (latest version at the moment)

**Comment by rsmckinney** [comment]
Fix available with release 2024.1.5. Thanks for reporting this!

**Comment by EotT123** [comment]
The first error is resolved (thanks!), but the second isn't. 
The following extension class (even without any methods) fails with a non-descriptive error "`Compilation failure`":

```
package extensions.java.awt.Component;

import manifold.ext.rt.api.Extension;

@Extension
public class ComponentExt {

}
```

It's not for all `java.awt.*` classes, as extending `java.awt.Container` works fine.

**Comment by rsmckinney** [comment]
Fix available with release 2024.1.6
